db/app.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express"),
+  app = express();
+const { getAllCategories } = require("./controllers/controller.js");
+
+app.use(express.json());
```

Comment:
I can see you're thinking ahead but this line of code will only be needed to access the body in a post/patch request so we don't need it yet. Best practise is to only have code that is needed for this specific pull request.

Reply:
aaaah ok i will keep that in mind